src/pages/home/report/ReportActionItemSingle.js

```diff
@@ -228,6 +238,14 @@ function ReportActionItemSingle(props) {
                                 />
                             ))}
                         </PressableWithoutFeedback>
+                        {!!hasEmojiStatus && (
```

Comment:
Use Boolean(hasEmojiStatus)

Reply:
Using Boolean is more straightforward here, instead of double not.  We should capture this in the style guide, I noticed using `Boolean()` for type conversion in Migration jobs.

Reply:
It looks like we use `Boolean` a lot more than `!!` in the code so let's just go with that

Reply:
👍🏻